app/models/enterprise.rb

```diff
@@ -420,6 +428,10 @@ def strip_url(url)
     url&.sub(%r{(https?://)?}, '')
   end
 
+  def correct_social_url(url)
+    url&.include?("@") ? url.delete!("@") : url
```

Comment:
This can be simplified:
```suggestion
    url&.delete("@")
```

* It deletes the `@` only when present.
* The return value is the same.
* It doesn't modify the original string. Displaying should not change data.

Reply:
You have a point🙂🙏 will do so asap